### PR TITLE
Portfolio : Mobile-version-setup

### PR DESCRIPTION
**Pull request by Hirwacardinalloic**
## Description 
In this PR,  we  create  a responsive mobile version 

### General requirements  
 - [x]  no linter errors.
 - [x] Correct  **Github flow**. 
 - [x] professional documentation . 
 
 ### Project  requirements, We: 
 - [x]  Added Mobile version with HTML and CSS

 # Happy reviewing 😃

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -70,3 +71,250 @@ body {
...
+.scroll {
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  width: 100%;
+  padding: 100px 0;
+  background-color: #f7f7f9;
+}
```
Comment (micheaol) [inline style.css]: Please ensure that the main container of the projects has a display: grid according to project requirements.

File: style.css [hunk 2]
```diff
@@ -70,3 +71,250 @@ body {
...
+
+.box-contain {
+  border: 1px solid #7a869a;
+  width: 90%;
+  margin: 0 auto;
+  border-radius: 16px;
+  background-color: #fff;
+  list-style: none;
+  margin-bottom: 50px;
```
Comment (micheaol) [inline style.css]: Please ensure that for each project the content is aligned with the flexbox according to project requirement

File: style.css [hunk 3]
```diff
@@ -70,3 +71,250 @@ body {
...
+  border-radius: 6px;
+}
+
+.about-me {
+  width: 90%;
+  margin: 0 auto;
+  overflow: hidden;
+  background-color: #fff;
+  border-top-right-radius: 5rem;
+  padding-top: 5rem;
```
Comment (micheaol) [inline style.css]: Please ensure that the `about section` content is aligned with flexbox according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Hirwacardinalloic , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Kakalanp — APPROVED** [review]
# STATUS: APPROVED :green_circle: 

### Your project is approved @Hirwacardinalloic! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**